weasel/info.py

```diff
@@ -0,0 +1,11 @@
+NAME = "weasel"
+HELP = """Command-line interface for weasel projects and templates.
+You'd typically start by cloning a project template to a local directory and
+fetching its assets like datasets etc. See the project's {PROJECT_FILE} for the
+available commands.
+
+DOCS: https://github.com/explosion/weasel
```

Comment:
Do we plan a proper website-based documentation?

Reply:
I don't think that's planned at the moment. I think most of the functionality will continue to covered in the spaCy docs, with coverage in the README here as well. It might also make sense to have separate, more structured docs here.